python/pylibcudf/pylibcudf/strings/convert/convert_ipv4.pyx

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+from libcpp.memory cimport unique_ptr
+from libcpp.utility cimport move
+from pylibcudf.column cimport Column
+from pylibcudf.libcudf.column.column cimport column
+from pylibcudf.libcudf.strings.convert cimport convert_ipv4 as cpp_convert_ipv4
+
+
+cpdef Column ipv4_to_integers(Column input):
+    """
+    Converts IPv4 addresses into integers.
+
+    For details, see cpp:func:`cudf::strings::ipv4_to_integers`
+
+    Parameters
+    ----------
+    input : Column
+        Strings instance for this operation
+
+    Returns
+    -------
+    Column
+        New int32 column converted from strings.
+    """
+    cdef unique_ptr[column] c_result
+
+    with nogil:
+        c_result = move(
+            cpp_convert_ipv4.ipv4_to_integers(
+                input.view()
+            )
+        )
+
+    return Column.from_libcudf(move(c_result))
+
+
+cpdef Column integers_to_ipv4(Column integers):
+    """
+    Converts integers into IPv4 addresses as strings.
+
+    For details, see cpp:func:`cudf::strings::integers_to_ipv4`
+
+    Parameters
+    ----------
+    integers : Column
+        Integer (uint32) column to convert.
```

Comment:
This isays uint32 whereas ipv4_to_integers above says it produces int32. Should these be the same, or is one really unsigned while the other is signed?

Reply:
Thanks, this was a documentation mistake on my end. `ipv4_to_integers` should say it returns uint32 integers (matching the cpp docstring and the unit test in this PR)

Reply:
OK cool yes please match those when you resolve conflicts, then you are good to go.